docs/metrics/prometheus/list.md

```diff
@@ -303,3 +303,109 @@ For deployments with [Site Replication](https://min.io/docs/minio/linux/operatio
 | `minio_bucket_requests_total`                     | Total number of S3 requests on a bucket.                                        |
 | `minio_bucket_requests_canceled_total`            | Total number S3 requests canceled by the client.                                |
 | `minio_bucket_requests_ttfb_seconds_distribution` | Distribution of time to first byte across API calls per bucket.                 |
+
+# Resource Metrics
+
+MinIO collects the following resource metrics at the node level.
```

Comment:
```suggestion
MinIO collects the following resource metrics at the cluster level.
```

Assuming this is cluster aggregated? if not, ignore.

Reply:
@anjalshireesh to confirm, should users issue this metrics endpoint per-node then?

Reply:
@ravindk89 no, the metrics are returned for all nodes (one entry per node, not aggregated). So the metrics are per-node, but one call returns metrics of all the nodes.